docs/manual/troubleshooting.tex

```diff
@@ -930,7 +930,7 @@ \subsection{Build the StubParser\label{stubparser-building}}
 \begin{Verbatim}
   cd $JSR308/stubparser
   mvn package -Dmaven.test.skip=true
-  cp -i ./javaparser-core/target/javaparser-core-3.3.4-SNAPSHOT.jar stubparser.jar
+  cp -i ./javaparser-core/target/stubparser.jar stubparser.jar
```

Comment:
How about we remove these copies: we only need a version independent path to the jar file. We can just use `./javaparser-core/target/stubparser.jar` instead of the top-level `stubparser.jar`. There is no benefit to copying it now that there is a consistent file name.
It should be enough to update https://github.com/typetools/checker-framework/blob/master/build-common.properties#L42 and then we can just remove these copy commands.